mapfishapp/src/main/webapp/app/addons/osmeditors/config.json

```diff
@@ -15,12 +15,14 @@
         "en": "Edit in OSM",
         "fr": "Édition dans OSM",
         "es": "Edición en OSM",
-        "de": "Bearbeiten in OSM"
+        "de": "Bearbeiten in OSM",
+		  "nl": "Bewerken in OSM"
```

Comment:
Please mind the space characters used (tabs ?)

Reply:
Fixed (tabs replaced by spaces)